### PR TITLE
added gitignore

### PR DESCRIPTION
**Pull request by nilsffcg**

### CODE REVIEW

**Comment by stakater-user** [comment]
@nilsffcg Yikes! You better fix it before anyone else finds out! [Build 1](https://jenkins-delivery.workshop.stakater.com/job/NordMart%20Application/job/stakater-nordmart-review/job/PR-1/1/) has Failed!

**Comment by stakater-user** [comment]
@nilsffcg Image is available for testing. `docker pull docker-delivery.workshop.stakater.com:443/stakater-lab/review:0.0.0-PR-1-2-SNAPSHOT`

**Comment by stakater-user** [comment]
@nilsffcg Yikes! You better fix it before anyone else finds out! [Build 1](https://jenkins-delivery.workshop.stakater.com/job/NordMart%20Application/job/stakater-nordmart-review/job/PR-1/1/) has Failed!

**Comment by stakater-user** [comment]
@nilsffcg Image is available for testing. `docker pull docker-delivery.workshop.stakater.com:443/stakater-lab/review:0.0.0-PR-1-2-SNAPSHOT`

**Comment by stakater-user** [comment]
@nilsffcg Image is available for testing. `docker pull docker-delivery.workshop.stakater.com:443/stakater-lab/review:0.0.0-PR-1-3-SNAPSHOT`

**Comment by stakater-user** [comment]
@nilsffcg Image is available for testing. `docker pull docker-delivery.workshop.stakater.com:443/stakater-lab/review:0.0.0-PR-1-4-SNAPSHOT`

**Comment by stakater-user** [comment]
@nilsffcg Image is available for testing. `docker pull docker-delivery.workshop.stakater.com:443/stakater-lab/review:0.0.0-PR-1-5-SNAPSHOT`

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -0,0 +1,6 @@
+spring.application.name=review-service
+spring.jersey.application-path=/api
+application.mode=dev
+
+spring.data.mongodb.database=nordmartDB
```
Comment (aliartiza75) [inline src/main/resources/application.properties]: Kindly change it to

{DB_NAME:nordmartDB}

DB_NAME is the environment variable and if it is not provided then the default value nordmartDB will be used. Lets keeps at the dynamic configuration part in this manner

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+spring.data.mongodb.uri=mongodb://mongodb:27017
```
Comment (aliartiza75) [inline src/main/resources/application.properties]: kindly change it to this format: mongodb://{MONGO_HOST:mongodb:27017}

**Comment by stakater-user** [comment]
@nilsffcg Image is available for testing. `docker pull docker-delivery.workshop.stakater.com:443/stakater-lab/review:0.0.0-PR-1-6-SNAPSHOT`

**Comment by stakater-user** [comment]
@nilsffcg Image is available for testing. `docker pull docker-delivery.workshop.stakater.com:443/stakater-lab/review:0.0.0-PR-1-7-SNAPSHOT`

**Comment by stakater-user** [comment]
@nilsffcg Image is available for testing. `docker pull docker-delivery.workshop.stakater.com:443/stakater-lab/review:0.0.0-PR-1-8-SNAPSHOT`

**Comment by stakater-user** [comment]
@nilsffcg Image is available for testing. `docker pull docker-delivery.workshop.stakater.com:443/stakater-lab/review:0.0.0-PR-1-9-SNAPSHOT`

File: src/main/java/com/stakater/nordmart/model/Review.java [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    private String customerName = "";
+    private int rating = 3;
+    private String reviewText = "";
+    private Date dateTime;
+
+    public Review() {
+
+    }
+
+    public Review(String productId, String custumerName, int rating, String text) {
```
Comment (aliartiza75) [inline src/main/java/com/stakater/nordmart/model/Review.java]: @nilsffcg kindly fix the typo `custumerName` to `customerName`.

File: src/main/java/com/stakater/nordmart/service/ReviewService.java [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+import com.stakater.nordmart.model.Review;
+
+import java.util.List;
+
+public interface ReviewService {
+
+    public List<Review> getReviews(String productId);
+
+    public Review addReview(String productId, String costumerName, int rating, String text);
```
Comment (aliartiza75) [inline src/main/java/com/stakater/nordmart/service/ReviewService.java]: Kindly fix the typo here

**Comment by stakater-user** [comment]
@nilsffcg Yikes! You better fix it before anyone else finds out! [Build 2](https://jenkins-delivery.workshop.stakater.com/job/Nordmart%20Apllications/job/stakater-nordmart-review/job/PR-1/2/) has Failed!

**Comment by stakater-user** [comment]
@nilsffcg Yikes! You better fix it before anyone else finds out! [Build 3](https://jenkins-delivery.workshop.stakater.com/job/Nordmart%20Apllications/job/stakater-nordmart-review/job/PR-1/3/) has Failed!

**Comment by stakater-user** [comment]
@nilsffcg Image is available for testing. `docker pull docker-delivery.workshop.stakater.com:443/stakater-lab/review:0.0.0-PR-1-4-SNAPSHOT`

**Comment by stakater-user** [comment]
@nilsffcg Image is available for testing. `docker pull docker-delivery.workshop.stakater.com:443/stakater-lab/review:0.0.0-PR-1-5-SNAPSHOT`

**Comment by stakater-user** [comment]
@nilsffcg Image is available for testing. `docker pull docker-delivery.cp-stakater.com:443/stakater-lab/review:0.0.0-PR-1-1-SNAPSHOT`

**Comment by stakater-user** [comment]
@nilsffcg Image is available for testing. `docker pull docker-delivery.cp-stakater.com:443/stakater-lab/review:0.0.0-PR-1-3-SNAPSHOT`

**Comment by stakater-user** [comment]
@nilsffcg Image is available for testing. `docker pull docker-delivery.cp-stakater.com:443/stakater-lab/review:0.0.0-PR-1-4-SNAPSHOT`

**Comment by stakater-user** [comment]
@nilsffcg Yikes! You better fix it before anyone else finds out! [Build 5](https://jenkins-delivery.cp-stakater.com/job/NordMart%20Applications/job/stakater-nordmart-review/job/PR-1/5/) has Failed!

**Comment by stakater-user** [comment]
@nilsffcg Image is available for testing. `docker pull docker-delivery.cp-stakater.com:443/stakater-lab/review:0.0.0-PR-1-6-SNAPSHOT`

**Comment by stakater-user** [comment]
@nilsffcg Image is available for testing. `docker pull docker-delivery.cp-stakater.com:443/stakater-lab/review:0.0.0-PR-1-7-SNAPSHOT`

**Comment by stakater-user** [comment]
@nilsffcg Yikes! You better fix it before anyone else finds out! [Build 8](https://jenkins-delivery.cp-stakater.com/job/NordMart%20Applications/job/stakater-nordmart-review/job/PR-1/8/) has Failed!

**Comment by stakater-user** [comment]
@nilsffcg Image is available for testing. `docker pull docker-delivery.cp-stakater.com:443/stakater-lab/review:0.0.0-PR-1-9-SNAPSHOT`

**Comment by stakater-user** [comment]
@nilsffcg Image is available for testing. `docker pull docker-delivery.cp-stakater.com:443/stakater-lab/review:0.0.0-PR-1-10-SNAPSHOT`

**Comment by stakater-user** [comment]
@nilsffcg Image is available for testing. `docker pull docker-delivery.cp-stakater.com:443/stakater-lab/review:0.0.0-PR-1-11-SNAPSHOT`

**Comment by stakater-user** [comment]
@nilsffcg Yikes! You better fix it before anyone else finds out! [Build 12](https://jenkins-delivery.cp-stakater.com/job/NordMart%20Applications/job/stakater-nordmart-review/job/PR-1/12/) has Failed!

**Comment by stakater-user** [comment]
@nilsffcg Image is available for testing. `docker pull docker-delivery.cp-stakater.com:443/stakater-lab/review:0.0.0-PR-1-13-SNAPSHOT`

**Comment by stakater-user** [comment]
@nilsffcg Yikes! You better fix it before anyone else finds out! [Build 14](https://jenkins-delivery.cp-stakater.com/job/NordMart%20Applications/job/stakater-nordmart-review/job/PR-1/14/) has Failed!

**Comment by stakater-user** [comment]
@nilsffcg Image is available for testing. `docker pull docker-delivery.cp-stakater.com:443/stakater-lab/review:0.0.0-PR-1-15-SNAPSHOT`

**Comment by stakater-user** [comment]
@nilsffcg Image is available for testing. `docker pull docker-delivery.cp-stakater.com:443/stakater-lab/review:0.0.0-PR-1-16-SNAPSHOT`

**Comment by stakater-user** [comment]
@nilsffcg Image is available for testing. `docker pull docker-delivery.cp-stakater.com:443/stakater-lab/review:0.0.0-PR-1-17-SNAPSHOT`

**Comment by stakater-user** [comment]
@nilsffcg Image is available for testing. `docker pull docker-delivery.cp-stakater.com:443/stakater-lab/review:0.0.0-PR-1-18-SNAPSHOT`

**Comment by stakater-user** [comment]
@nilsffcg Image is available for testing. `docker pull docker-delivery.cp-stakater.com:443/stakater-lab/review:0.0.0-PR-1-19-SNAPSHOT`

**Comment by stakater-user** [comment]
@nilsffcg Image is available for testing. `docker pull docker-delivery.cp-stakater.com:443/stakater-lab/review:0.0.0-PR-1-20-SNAPSHOT`

**Comment by stakater-user** [comment]
@nilsffcg Image is available for testing. `docker pull docker-delivery.cp-stakater.com:443/stakater-lab/review:0.0.0-PR-1-21-SNAPSHOT`

**Pull request closed without merging**